docs/src/main/asciidoc/jms.adoc

```diff
@@ -290,6 +319,83 @@ You can build the native executable with:
 ./mvnw package -Pnative
 ----
 
-== Configuration Reference
+Or, if you dont have GraalVM installed, you can instead use Docker to build the native executable using:
+[source, shell]
+----
+./mvnw package -Pnative -Dquarkus.native.container-build=true
+----
+
+and then run with:
+
+[source, shell]
+----
+./target/jms-quickstart-1.0-SNAPSHOT-runner
+----
+
+Open `http://localhost:8080/prices.html` in your browser.
+
+'''
```

Comment:
What is this?

Reply:
A line separator between the two outer client sections to make it a little more obvious this is the end of the steps.